### ISSUE TITLE
CreateDynamicImport(0) should not parse comments

### ISSUE THREAD

**Issue by michkowalczuk** [neslib/Chet#33]
Static (default) mode in Chet is more stable than dynamic.
Example:
In static mode Chet returns code with comments correctly moved from C header.
In my case this comment contains string:
```
/**
The outer loop to be set for the face. If the
function succeeds, this loop will be deallocated.
*/
```
After using `CreateDynamicImport(0)` Chet generates the following method, which is a bug:
`succeeds := GetProcAddress(aDLLHandle, 'succeeds');`

**Comment by jarroddavis68** [comment]
Yup, I'm working on fixing this. Big update to post processing coming. 

Another issue I recently discovered too is advanced records, it will try to do the same. So I have to figure out these corner cases.

**Comment by michkowalczuk** [comment]
Please update this thread once you've fixed it.

**Comment by jarroddavis68** [comment]
> Please update this thread once you've fixed it.

Will do.
